Win/zabbix_agentd.conf.d/userparameter_smartctl.conf

```
@@ -1 +1 @@
-UserParameter=smartctl.discovery[*], python "C:\zabbix-agent\scripts\smartctl-lld.py" "$1" "$2"
+UserParameter=smartctl.discovery[*], python "C:\I\dont\know\the\correct\path\zabbix-smartctl.exe" "$1" "$2"
```

```suggestion
UserParameter=smartctl.discovery[*], zabbix-smartctl.exe "$1" "$2"
```
Fortunately, it's in PATH.